app/services/air_now/process_measurements.rb

```diff
@@ -0,0 +1,51 @@
+class AirNow::ProcessMeasurements
+  def initialize(measurements)
+    @measurements = measurements
+  end
+
+  def call
+    @measurements.each_with_object([]) do |measurement, processed_measurements|
+      next unless wanted_sensor_name?(measurement)
+
+      measurement[:parameter] = normalize_sensor_name(measurement)
+      processed_measurements << create_saveable_object(measurement)
+    end
+  end
+
+  private
+
+  attr_reader :measurements
+
+  def wanted_sensor_name?(measurement)
+    ['PM2.5', 'O3', 'NO2', 'OZONE'].include?(measurement[:parameter])
+  end
+
+  def normalize_sensor_name(measurement)
+    measurement[:parameter] == 'OZONE' ? 'O3' : measurement[:parameter]
+  end
```

Comment:
I would pass `parameter` instead of the whole `measurement`. The other methods are written like this and I think it's quicker to understand.